packages/block-editor/src/components/inserter/menu.js

```diff
@@ -457,6 +490,10 @@ export default compose(
 			__experimentalFetchReusableBlocks: fetchReusableBlocks,
 		} = dispatch( 'core/editor' );
 
+		const {
+			openModal,
+		} = dispatch( 'core/edit-post' );
```

Comment:
Same here, we're not allowed to use actions from the ` core/edit-post` module in this package.